1_task/1_task.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Created by Alex on 17.03.2017.
+ */
+
+function addition(a, b) {
+    return +a + +b;
+}
+
+function subtraction(a, b) {
+    return a - b;
```

Comment:
you mean I must use 2 spaces instead 4(it's default doing Idea)?
// bad
function() {
∙∙∙∙var name;
}

// bad
function() {
∙var name;
}

// good
function() {
∙∙var name;
}